ios/Classes/Login Button/FacebookLoginButtonProxy.m

```diff
@@ -18,21 +18,33 @@ - (NSString *)apiName
   return @"Ti.Facebook.LoginButton";
 }
 
+- (NSArray *_Nullable)publishPermissions
+{
+  return [[(FacebookLoginButton *)self.view loginButton] publishPermissions];
```

Comment:
To avoid repeated casting a small internal utility method might be useful. Something like

```objc
- (FacebookLoginButton *)loginButton {
  return [(FacebookLoginButton *)self.view loginButton]
}
```

Reply:
Good idea. Updated!